Modules/Numerics/Optimizers/include/itkFRPROptimizer.h

```diff
@@ -70,6 +70,9 @@ extern ITKOptimizers_EXPORT std::ostream &
 class ITKOptimizers_EXPORT FRPROptimizer : public PowellOptimizer
 {
 public:
+  /** Copy constructor */
+  FRPROptimizer(const FRPROptimizer &) = delete;
```

Comment:
assignment operator shoulld be deleted here as well, so use DISALLOW.

Reply:
On second thought.  Try to use "DISALLOW" but then check to see if the operator= assignment is used anywhere.  We may need the assignment operator.

Reply:
No build errors occurred, so DISALLOW works here.